android/src/main/java/expo/modules/paywallsdk/HeliumPaywallSdkModule.kt

```
@@ -250,7 +251,21 @@ class HeliumPaywallSdkModule : Module() {
         Helium.config.heliumPaywallDelegate = delegate
         customAPIEndpoint?.let { Helium.config.customApiEndpoint = it }
 
-        setupFallbackBundle(context, fallbackBundleUrlString, fallbackBundleString)
+        // Pass fallback JSON to native SDK
+        val fallbackJsonString: String? = when {
+          fallbackBundleUrlString != null -> {
+            try {
+              val sourceFile = java.io.File(java.net.URI.create(fallbackBundleUrlString))
+              if (sourceFile.exists()) sourceFile.readText() else null
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)